client/src/components/helpers/CurrentChallengeLink.js

```diff
@@ -11,7 +11,8 @@ const currentChallengeApi = '/challenges/current-challenge';
 
 const propTypes = {
   children: PropTypes.any,
-  hardGoTo: PropTypes.func.isRequired
+  hardGoTo: PropTypes.func.isRequired,
+  isLargeBtn: PropTypes.string
```

Comment:
```suggestion
  isLargeBtn: PropTypes.bool
```

Reply:
Yeah....I noticed it right now...fixed it